apps/s_socket.c

```diff
@@ -74,6 +76,15 @@ int init_client(int *sock, const char *host, const char *port,
         return 0;
     }
 
+    if (local != NULL) {
+        ret = BIO_lookup (local, NULL, BIO_LOOKUP_CLIENT, family, type,
```

Comment:
Style nit. Style guide says no space between subroutine name and parenthesis.

Reply:
Fixed.